### PR TITLE
Phasic strilka ammo no longer printable

### PR DESCRIPTION
**Pull request by Swiftfeather**
## About The Pull Request

Title

## Why It's Good For The Game

There's no reason people should be running around with it

## Proof Of Testing

I didn't test it

## Changelog

:cl:
del: Phasic strilka ammo is no longer printable.
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/projectiles/ammunition/ballistic/rifle.dm [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/obj/item/ammo_casing/strilka310/phasic
+	can_be_printed = FALSE
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/modules/projectiles/ammunition/ballistic/rifle.dm:2]: Why make this a modular file? isn't this ammo Skyrat?

Comment (Swiftfeather) [reply modular_zubbers/code/modules/projectiles/ammunition/ballistic/rifle.dm:2]: It's tg ammo https://github.com/Bubberstation/Bubberstation/blob/1c9d1ac9a1796f5830e4bdd4ba98f4f43bb25ce1/code/modules/projectiles/ammunition/ballistic/rifle.dm#L23

Comment (Swiftfeather) [reply modular_zubbers/code/modules/projectiles/ammunition/ballistic/rifle.dm:2]: WAIT ARE THESE UNDOCUMENTED SKYRAT CHANGES???????

Comment (Paxilmaniac) [reply modular_zubbers/code/modules/projectiles/ammunition/ballistic/rifle.dm:2]: Coming to a pr that was merged over a year ago to complain is crazy, hop off my dick bruh.

**Comment by StrangeWeirdKitten** [comment]
I was going to yell at swift for talking shit in a year old PR on TG, but then I was made aware that you're going into other gun PRs on our github, talking shit, because we touched things you coded, on a server you don't even play on, or a github you don't contribute too. Litterally rent free.

Anyway, you don't have to worry about that anymore. You can move on doing whatever you do now and we will continue running our server, thanks.

**Review by lessthnthree — APPROVED** [review]

**Pull request merged**